src/Response/SapiStreamEmitter.php

```diff
@@ -24,9 +24,10 @@ class SapiStreamEmitter implements EmitterInterface
      * body content via the output buffer.
      *
      * @param ResponseInterface $response
+     * @param null|int $maxBufferLevel Maximum output buffering level to unwrap.
      * @param int $maxBufferLength Maximum output buffering size for each iteration
      */
-    public function emit(ResponseInterface $response, $maxBufferLength = 8192)
+    public function emit(ResponseInterface $response, $maxBufferLevel = null, $maxBufferLength = 8192)
```

Comment:
For backwards compatibility, the last two arguments **MUST** be swapped in order. Otherwise, somebody providing a `$maxBufferLength` value with the existing signature will now be providing a `$maxBufferLevel` value instead, which could have unexpected repurcussions.